### PR TITLE
Login hide and show password 

### PR DESCRIPTION
**Pull request by Tanjum786**
https://user-images.githubusercontent.com/84380051/162623688-22435a48-59a3-46d9-8f82-5b2c0928b8a0.mp4

### CODE REVIEW

File: src/backend/db/products.js [hunk 1]
```diff
@@ -31,6 +31,7 @@ export const products = [
     title: "Adidas Men's T-Shirts",
     img: productimg1,
     price: 499,
+    Quantity: 1,
```
Comment (tanishq20) [inline src/backend/db/products.js]: This quantity part can be handled by the Mockbee backend also. Refer to docs and API controllers for more info.

File: src/pages/Authentication/Singup.jsx [hunk 2]
```diff
@@ -1,18 +1,20 @@
 import React from "react";
 import { Link } from "react-router-dom";
 import { Footer, Navbar } from "../../components";
+import { useloginsignup } from "../../hooks/context/loginSignup-context";
 import "./Authentication.css";
 
 export const Singup = () => {
+  const {showpassword,showpasswordFun}=useloginsignup();
```
Comment (tanishq20) [inline src/pages/Authentication/Singup.jsx]: You can use a proper naming convention for more readability of code. Eg: showPasswordFun (Apply this in all)

**Review by tanishq20 — APPROVED** [review]
Code looks clean. Suggested some minor changes.

File: src/hooks/context/Reducer/CartRedcerFun.jsx [hunk 3]
```diff
@@ -0,0 +1,109 @@
+export const CartRedcerFun = (cartstate, cartaction) => {
```
Comment (vaibhav-mahalle) [inline src/hooks/context/Reducer/CartRedcerFun.jsx]: you can name it CartReducerFuc

**Review by vaibhav-mahalle — APPROVED** [review]
suggested a minor change everything else looks good.

**Pull request merged**